server-pro/Dockerfile

```
@@ -78,8 +78,8 @@ RUN chmod +x /usr/local/bin/startup.sh
 COPY conf/* /etc/rstudio/
 
 # Install RStudio Server Pro --------------------------------------------------#
-ARG RSP_VERSION=1.3.1093-1
-ARG RSP_DOWNLOAD_URL=https://download2.rstudio.org/server/bionic/amd64
+ARG RSP_VERSION=1.4.1103-3
+ARG RSP_DOWNLOAD_URL=https://download2.rstudio.org/server/xenial/amd64
```

```suggestion
ARG RSP_DOWNLOAD_URL=https://download2.rstudio.org/server/bionic/amd64
```
I _think_ this should still be bionic (it doesn't get used in the build process)